skin/light/light.css

```diff
@@ -241,6 +241,10 @@
   border-left: 4px solid #0996f8 !important;
 }
 
+.tabbrowser-tab[pinned] + .tabbrowser-tab:not([pinned]) {
+  border-top: 1px solid black !important;
+}
```

Comment:
Yeah, the border should really stay where it is…  Also when you create a new tab, the border disappears and fades in, but it shouldn't.  :(